### DISCUSSION TITLE
Switch back to normal OpenWRT

### DISCUSSION THREAD

**Discussion by heinzguenter** [xchwarze/wifi-pineapple-cloner#38] [Q&A] [answered]
Hello, sorry if this is a dumb question, but I am new to the openwrt buissness. I want to change my mango(MT300NV2) router back to plain openwrt, but how do I do it? Can i just use the update function in the webUi or is there any other way?

**Comment by Jopeke** [comment, marked as answer]
See Debricking gl-mt300n_v2 on OpenWRT website
